uint16 start
uint16 range
uint8 direction
uint16 crc